test/framework/check_setup.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/test/config"
+)
+
+var verifications = []checker{
+	checkKubernetesAndDomainSuffix(),
```

Comment:
In order to add new checkers, it would be just a matter to add a new function here.

Reply:
couldn't we just give the reference to the method instead of calling a method returning a `func` ?
like we do in the steps

Reply:
I thought adding the checker type function was clearer and this is used in other places within the Operator source. However, not a big deal. I've already updated to use the approaches in the steps.